test.sh

```diff
@@ -58,6 +58,23 @@ cd $REPO_ROOT_DIR
 
 # Get a CCM cluster if not already configured (see available settings in dcos-commons/tools/README.md):
 if [ -z "$CLUSTER_URL" ]; then
+    if [ "$SECURITY" = "strict" ]; then
```

Comment:
I don't agree.

run_tests.py accepts a running cluster, and needs to be able to talk to it, and people use run_tests.py directly already.

test.sh believes it needs to create a cluster, and launch_ccm_cluster.py unfortunately just  launch a cluster, it launches it and possibly adds a service account via use of the dcos command line.

These needs are distinct, and while restructuring the scripts so there's a tool that adds a dcos command line to the path for general use is worthwhile, there are opportunities for race conditions with a stable name, and fixing one problem can't require fixing all problems at once.

In short, yeah definitely I'll be doing that but not now to make smoke test work at all.